flake.nix

```diff
@@ -59,12 +57,11 @@
           go = final.go_1_17;
         };
         bundle-exe = import nix-bundle-exe { pkgs = final; };
-        bundle-exe-tarball = drv:
-          let bundle = final.bundle-exe drv;
-          in
-          final.runCommand bundle.name { } ''
-            "${final.gnutar}/bin/tar" cfzhv $out -C ${bundle} .
-          '';
+        make-tarball = drv: with final; runCommand drv.name { } ''
+          "${gnutar}/bin/tar" cfv - -C ${drv} \
+            --owner=0 --group=0 --mode=u+rw,uga+r --hard-dereference . \
```

Comment:
don't follow symbolic links.
don't leak the building user in the file attributes.
keep the normal file permissions.
follow hard links.

Reply:
maybe good to put this as a comment directly in the source code?

Reply:
sounds good, done.